twitterscraper/query.py

```diff
@@ -32,7 +32,7 @@ def query_single_page(url, html_response=True, retry=10):
 
     try:
         response = requests.get(url, headers=headers)
-        if resonse.status_code == 200:
+        if response.status_code == 200:
```

Comment:
guess you made a typo in the first commit and corrected it in the second? I'd either `reset` back to master and make two clean commits or at least squash them, this makes the history rather confusing.